tests/e2e/scripts/rancher.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+node_ip=$1
+
+echo  "Give K3s time to startup"
+sleep 15
+
+cat << EOF > rancher.yaml
```

Comment:
If this is running on the node itself, can we let the deploy controller pick it up?

```suggestion
cat << EOF > /var/lib/rancher/k3s/server/manifests/rancher.yaml
```